### PR TITLE
ADAPT-559 Reorganized file structure for cards and resolved line-height issue

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Reorganized faux and icon cards style files
- Added partial card file for h2 and p tag line height

# Review By (Date)
- asap

# Urgency
- low

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Compare Icon Card with WYSIWYG `<h2>` and `<p>` tags.

Line height should be:
Mobile: 22.4px
Standard: 35.7px

# Associated Issues and/or People
- [ADAPT-559](https://stanfordits.atlassian.net/browse/ADAPT-559)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by rebeccahongsf** [comment]
I've removed `icon-card--single-in-row` since it contained font-sizing that should be applied to all with the `.su-card` class. 
Font and line-height styles for `.su-card` can be found within `_card.scss` partial.

**Review by katrialesser — CHANGES_REQUESTED** [review]
rebuild to resolve css conflict, looks great, except we shouldn't have the @import line for the utilities. Those lines aren't needed now that we have done the webpack reorg.

File: src/scss/components/card/faux-cards/_faux-card--cta-list.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @charset "UTF-8";
+@import "../../../utilities/index";
```
Comment (katrialesser) [inline src/scss/components/card/faux-cards/_faux-card--cta-list.scss]: remove - not needed now with the webpack reorg method.

File: src/scss/components/card/faux-cards/_faux-card--fonts.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 @charset "UTF-8";
+@import "../../../utilities/index";
```
Comment (katrialesser) [inline src/scss/components/card/faux-cards/_faux-card--fonts.scss]: shouldn't be needed.

File: src/scss/components/card/faux-cards/_faux-card--horizontal-general.scss [hunk 3]
```diff
@@ -1,4 +1,5 @@
 @charset "UTF-8";
+@import "../../../utilities/index";
```
Comment (katrialesser) [inline src/scss/components/card/faux-cards/_faux-card--horizontal-general.scss]: shouldn't be needed.

File: src/scss/components/card/faux-cards/_faux-card--horizontal-min-circle.scss [hunk 4]
```diff
@@ -1,4 +1,5 @@
 @charset "UTF-8";
+@import "../../../utilities/index";
```
Comment (katrialesser) [inline src/scss/components/card/faux-cards/_faux-card--horizontal-min-circle.scss]: shouldn't be needed.

File: src/scss/components/card/faux-cards/_faux-card--horizontal-min-square.scss [hunk 5]
```diff
@@ -1,4 +1,5 @@
 @charset "UTF-8";
+@import "../../../utilities/index";
```
Comment (katrialesser) [inline src/scss/components/card/faux-cards/_faux-card--horizontal-min-square.scss]: shouldn't be needed.

File: src/scss/components/card/faux-cards/_faux-card--layout--media-text-area.scss [hunk 6]
```diff
@@ -1,4 +1,5 @@
 @charset "UTF-8";
+@import "../../../utilities/index";
```
Comment (katrialesser) [inline src/scss/components/card/faux-cards/_faux-card--layout--media-text-area.scss]: shouldn't be needed.

File: src/scss/components/card/faux-cards/_faux-card--vertical-general.scss [hunk 7]
```diff
@@ -1,4 +1,5 @@
 @charset "UTF-8";
+@import "../../../utilities/index";
```
Comment (katrialesser) [inline src/scss/components/card/faux-cards/_faux-card--vertical-general.scss]: shouldn't be needed.

File: src/scss/components/card/faux-cards/_media-caption-text-area-layout.scss [hunk 8]
```diff
@@ -1,4 +1,5 @@
 @charset "UTF-8";
+@import "../../../utilities/index";
```
Comment (katrialesser) [inline src/scss/components/card/faux-cards/_media-caption-text-area-layout.scss]: shouldn't be needed.

File: src/scss/components/card/icon-cards/_icon-card.scss [hunk 9]
```diff
@@ -1,5 +1,5 @@
 @charset "UTF-8";
-@import "../../utilities/index";
+@import "../../../utilities/index";
```
Comment (katrialesser) [inline src/scss/components/card/icon-cards/_icon-card.scss]: shouldn't be needed.

File: src/scss/components/index.scss [hunk 10]
```diff
@@ -17,6 +17,9 @@
 // .component-name--variant
 // .component-name__sub-object
 
+// Config and supporting scss.
+//@import '../config/index';
```
Comment (katrialesser) [inline src/scss/components/index.scss]: Remove comments.

**Review by katrialesser — CHANGES_REQUESTED** [review]

**Review by katrialesser — APPROVED** [review]

**Pull request merged**